src/features/features/member-count.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (client) => {
+    const channelId = '';
```

Comment:
Please add this to the `Guild.model.js` so this can be saved and edited using the dashboard

Reply:
Dont push this feature yet as it is not compatible with multiple guilds. Tmw I will work on this